content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -328,14 +328,24 @@ before liveness or readiness probes are initiated. Defaults to 0 seconds. Minimu
 * `periodSeconds`: How often (in seconds) to perform the probe. Default to 10
 seconds. Minimum value is 1.
 * `timeoutSeconds`: Number of seconds after which the probe times out. Defaults
-to 1 second. Minimum value is 1.
+to 1 second. Minimum value is 1. For exec probes, the process inside the container
+may keep running even after probe returned failure because of the timeout. Incorrect
```

Comment:
Since this only applies for dockershim is it worthwhile to mention that?

Reply:
as much as I want to migrate everybody out of DockerShim - there are still a lot of customers on DockerShim. Perhaps I can clarify that this only applies to DockerShim. I think this issue is worth mentioning so there would be no surprises.